Automattic/Co-Authors-Plus
# FIX: Correct unit tests when running WP 5.7

Fixes #458 

**Changes**

Correct two unit tests that are causing an error when running under WP 5.7.

**Steps to test**

1. Check out this PR
2. Run `phpunit` in the plugin directory

**Notes**

Testing this PR requires a temporary WordPress installation. A temporary WordPress installation can be created with the following steps:

1. Install MariaDB via `brew install mariadb`
2. Start MariaDB as service via `brew services start mariadb`
3. Go to plugin directory
4. Run `./bin/install-wp-tests.sh co_authors_plus root root`

Please note that this PR only resolves the problem that appears when using WordPress 5.7, the latest available version. It does not resolve the theme-related failed test case when using WordPress 5.3.

Please also note that the previous featured image was an artificial file and not a real image file. This caused a problem in the Twenty Twenty-One theme as this artificial file does not have any metadata such as a height or a width. Therefore, I've replaced that artificial file with a real image dummy file.
